backend/src/Controller/Order/OrderController.php

```diff
@@ -93,6 +94,17 @@ public function __construct(SerializerInterface $serializer, AutoMapping $autoMa
      *      )
      *   )
      * )
+     * 
+     * or
+     *
+     * @OA\Response(
+     *      response="default",
+     *      description="Return error.",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code", description="9151 or 9204"),
```

Comment:
If there are more than one potential status code, let us create a response annotation for each one please.